simplq/src/store/asyncActions/deleteQueue.js

```diff
@@ -11,11 +14,20 @@ const typePrefix = 'deleteQueue/action';
  */
 const useDeleteQueue = () => {
   const makeAuthedRequest = useMakeAuthedRequest();
+  const dispatch = useDispatch();
+  const history = useHistory();
 
   const deleteQueue = createAsyncThunk(typePrefix, async (arg) => {
     const { queueId } = arg;
     const authedRequest = makeAuthedRequest(RequestFactory.deleteQueue(queueId));
     const response = await authedRequest;
+    if (arg.goHome) {
+      history.push('/');
+    }
+    if (arg.popUp) {
+      dispatch(setInfoPopupMessage('Successfully deleted queue'));
+    }
+
```

Comment:
Added these to allow better reuse. This is used to delete queues on the homepage and the admin page.